testing/injections/timers_shim.js

```diff
@@ -0,0 +1,14 @@
+// ts file compiled js file somehow end up with cjs format but esbuild needs esm format for shims.
```

Comment:
isn't that controllable in tsconfig?

Reply:
it's controllable, but not on a single-file granularity (-> I didn't figure this out). I am not sure if it's time to move everything to esm because of this.